mpf/plugins/twitch/twitch_client.py

```diff
@@ -26,7 +26,10 @@ def __init__(self, machine, username, password, channel, loop):
         server = 'irc.chat.twitch.tv'
         port = 6667
         self.log.info('Connecting to ' + server + ' on port ' + str(port) + '...')
-        super().__init__([(server, port, 'oauth:' + password)], username, username)
+        super().__init__([(
+            server, port,
+            (password if password.lower().startswith('oauth:') else ('oauth:' + password))
```

Comment:
we could also have two parameters in config if that helps to make this more clear

Reply:
Most people are going to have the `oauth:` prefix on their passwords since that is what the token generator produces. I handle both because I did not copy the prefix and other people are going to make the same mistake.

Reply:
ok makes sense